### PR TITLE
(PC-23911) feat(ProfileTutorial): create 15 tutorial

### PR DESCRIPTION
**Pull request by marineshaw**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-23911

## Checklist

I have:

- [ ] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [ ] Written **unit tests** native (and web when implementation is different) for my feature.
- [ ] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

## Screenshots

**delete** *if no UI change*

| Platform         | Before | After |
| :--------------- | :----: | :---: |
| iOS              |        | image image|
| Android          |        | image image |
| Phone - Chrome   |        |       |
| Tablet - Chrome  |        |       |
| Desktop - Chrome |        |       |
| Desktop - Safari |        |       |

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/tutorial/components/CreditTimeline.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    type === 'onboarding' ? OnboardingCreditBlockTitle : ProfileTutorialCreditBlockTitle
+
+  return (
+    <Container>
+      {stepperProps.map((props, index) => {
+        const iconComponent = props.iconComponent ?? <GreyWarning />
+        if (props.creditStep === 'information')
+          return (
+            <InternalStep
+              key={'information ' + index}
```
Comment (Lucasbeneston) [inline src/features/tutorial/components/CreditTimeline.tsx:59]: Je pense que ça ne va rien changer à l'issue Sonar (si tu as fait ça pour ça)

File: src/features/tutorial/components/OnboardingTimeline.tsx [hunk 2]
```diff
@@ -1,172 +1,98 @@
...
 
-  const stepperProps = stepperPropsMapping.get(age)
+const DescriptionText = styled(Typo.Caption)(({ theme }) => ({
+  fontSize: theme.tabBar.fontSize,
+  lineHeight: getSpacingString(3),
+  color: theme.colors.greyDark,
+}))
+
+const CreditBlockContent = ({ description }: { description: string }) => {
   return (
```
Comment (Lucasbeneston) [inline src/features/tutorial/components/OnboardingTimeline.tsx:25]:
```suggestion
const CreditBlockContent = ({ description }: { description: string }) => ({
```

File: src/features/tutorial/components/OnboardingTimeline.tsx [hunk 3]
```diff
@@ -1,172 +1,98 @@
...
+      { creditStep: 16 },
+      { creditStep: 17 },
+      { creditStep: 'information', children: <StyledBody>Remise à 0 du crédit</StyledBody> },
       {
-        creditStep: 15,
-        variant: StepVariant.in_progress,
-        iconComponent: <AnimatedBicolorUnlock />,
+        creditStep: 18,
+        children: (
+          <CreditBlockContent description={'Tu auras 2 ans pour utiliser tes 300\u00a0€'} />
```
Comment (Lucasbeneston) [inline src/features/tutorial/components/OnboardingTimeline.tsx:50]:
```suggestion
          <CreditBlockContent description="Tu auras 2 ans pour utiliser tes 300&nbsp;€" />
```

File: src/features/tutorial/components/OnboardingTimeline.tsx [hunk 4]
```diff
@@ -1,172 +1,98 @@
...
+      { creditStep: 16 },
+      { creditStep: 17 },
+      { creditStep: 'information', children: <StyledBody>Remise à 0 du crédit</StyledBody> },
       {
-        creditStep: 17,
-        variant: StepVariant.in_progress,
-        iconComponent: <AnimatedBicolorUnlock />,
+        creditStep: 18,
+        children: (
+          <CreditBlockContent description={'Tu auras 2 ans pour utiliser tes 300\u00a0€'} />
```
Comment (Lucasbeneston) [inline src/features/tutorial/components/OnboardingTimeline.tsx:80]:
```suggestion
          <CreditBlockContent description="Tu auras 2 ans pour utiliser tes 300&nbsp;€" />
```

File: src/features/tutorial/components/OnboardingTimeline.tsx [hunk 5]
```diff
@@ -1,172 +1,98 @@
...
-      { creditStep: 17, variant: StepVariant.complete, iconComponent: <MediumGreyLock /> },
+      { creditStep: 15 },
+      { creditStep: 16 },
+      { creditStep: 17 },
       {
         creditStep: 18,
-        variant: StepVariant.in_progress,
-        iconComponent: <AnimatedBicolorUnlock />,
+        children: (
+          <CreditBlockContent description={'Tu auras 2 ans pour utiliser tes 300\u00a0€'} />
```
Comment (Lucasbeneston) [inline src/features/tutorial/components/OnboardingTimeline.tsx:94]:
```suggestion
          <CreditBlockContent description="Tu auras 2 ans pour utiliser tes 300&nbsp;€" />
```

File: src/features/tutorial/components/CreditBlock.stories.tsx [hunk 6]
```diff
@@ -46,38 +42,49 @@ const StyledTitle3 = styled(Typo.Title3)(({ theme }) => ({
...
 }
 
 export const withDescription = Template.bind({})
 withDescription.args = {
-  title: <Typo.Title3>300&nbsp;€</Typo.Title3>,
+  children: (
+    <React.Fragment>
+      <Typo.Title3>300&nbsp;€</Typo.Title3>
+      <Spacer.Column numberOfSpaces={2} />
+      <StyledBody>'Tu auras 2 ans pour utiliser tes 300\u00a0€'</StyledBody>
```
Comment (voisinhugo) [inline src/features/tutorial/components/CreditBlock.stories.tsx:78]: @marineshaw j'ai l'impression qu'ESLint meure à cause des apostrophes (qui devraient pas être, d'ailleurs)

**Review by Lucasbeneston — APPROVED** [review]
LGTM 👍

**Pull request merged**